nautobot/dcim/tests/test_forms.py

```diff
@@ -43,6 +45,15 @@ def setUpTestData(cls):
         cls.manufacturer = cls.device_type.manufacturer
         cls.platform = Platform.objects.filter(manufacturer=cls.device_type.manufacturer).first()
         cls.device_role = Role.objects.get_for_model(Device).first()
+        cls.software_version_contains_no_valid_image_for_device_type = SoftwareVersion.objects.create(
+            platform=cls.platform,
+            version="New version 1.0.0",
+            status=Status.objects.get_for_model(SoftwareVersion).first(),
+        )
+        cls.software_version = SoftwareVersion.objects.first()
+        cls.software_image_files = SoftwareImageFile.objects.exclude(
+            software_version=cls.software_version, default_image=True
+        )
```

Comment:
Is this logic right? Or should it be `.exclude(software_version=...).exclude(default_image=True)`?